code/script.js

```diff
@@ -1,19 +1,228 @@
 // Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Welcome to Javascript Restaurant! \nReady to start ordering? - Click 'OK' to begin.`
 )
 
-// Step 2 - Food choice
-// Your code goes here
+// Step 2 - Name
+const customerName = prompt(
+  `Hi! Welcome to our interactive food ordering experience. \nFirst and foremost, what is your name?`
+)
+
+// Step 3 - Food choice
+if (customerName == "") {
+  alert(
+    `Hmm... Looks like no name was entered, and I'm pretty sure you have a name. Please try again.`
+    )
+}
+
+else {
+  alert(
+    `Nice of you to pop by, ${customerName}! Now let's get you some food, shall we?`
+  )
+}
+
+const food = prompt(
+`What do you feel like eating today? \nChoose your preference by typing in the corresponding number.
```

Comment:
did you know? Inside your template literals, you can directly write your text in multiple lines without using a \n symbol to create a new line:

```
`What do you feel like eating today? 
Choose your preference by typing in the corresponding number`
```